src/scss/components/_listings.scss

```diff
@@ -90,179 +103,168 @@ $listing-borderRadius       : 4px;
                 width: 65px;
+                top: auto;
+                left: auto;
+                position: relative;
             }
         }
 
         .c-listing-item-imgItem {
             width: 100%;
             height: auto;
+            border-radius: $listing-img-borderRadius;
         }
 
         .c-listing-item-info {
             top: -4px;
             position: relative;
             color: $color-text;
-            padding-left: 56px;
-
-            @include media('>=tiny') {
-                padding-left: 66px;
-            }
+            padding-left: 70px;
 
             @include media('>wide') {
+                flex-grow: 1;
                 position: relative;
-                padding-left: 80px;
-                padding-right: 260px;
+                padding-left: 15px;
 
                 &:after {
                     content: '';
                     top: 4px;
                     width: 1px;
-                    right: 259px;
+                    right: -1px;
                     height: 100%;
                     display: block;
                     position: absolute;
                     background: $listing-borderColor;
-
-                    .c-listing--offline & {
-                        background: $listing-offlineBorderColor;
-                    }
                 }
             }
         }
 
-        .c-listing-item-title {
-            word-break: break-word;
-        }
-
-        // .c-listing-item-meta should be used as a container for listing labels, logos or offers (e.g. local legend logo or 20% off label)
-        .c-listing-item-meta {
-            top: -3px;
-            float: right;
-            margin-top: 0;
-            position: relative;
+        .c-listing-item-details {
+            width: 100%;
 
             @include media('>wide') {
-                top: auto;
-                width: 260px;
-                padding-bottom: 50px;
+                width: 460px;
+                margin-top: auto;
                 padding-left: spacing(x2);
                 border-left: 1px solid $listing-borderColor;
-
-                .c-listing--offline & {
-                    border-color: $listing-offlineBorderColor;
-                }
             }
         }
 
-        .c-listing-item-rating {
-            padding: 2px 0 3px;
+        .c-listing-item-detailsRow {
+            display : flex;
+            align-items : center;
+            margin-bottom: spacing() / 2;
+        }
 
-            .c-listing-item-ratingStars,
-            .c-listing-item-ratingText {
-                vertical-align: middle;
-            }
+        .c-listing-item-detailsRow-icon {
+            width: 20px;
+            height: 20px;
+            fill: $grey--dark;
         }
 
-        .c-listing-item-details {
-            width: 100%;
-            padding-left: 56px;
+        .c-listing-item-detailsRow-text {
+            margin-left: spacing();
+        }
+
+        .c-listing-item-promotion {
```

Comment:
What's the difference between this class and the `c-listing-item-promo`?

Reply:
the promotion is the tiles 20% off text. The promo is the alternative tile label "BTA winner 2018" but was labled promotion. 

I was blind! 

is `.c-listing-item-labelAlt` okay?

Reply:
Makes sense – so they're both promotions of different things. I guess the BTA one could be an `award` and be labelled like that?  The 20% should likely be labelled `promo`

Reply:
yeah - that sounds good. Will update now.

Reply:
fixed 😃 👍